### DISCUSSION TITLE
Assignable hotkeys to switch streams

### DISCUSSION THREAD

**Discussion by Peatote** [f1multiviewer/issue-tracker#252] [Feature requests]
Example: Pressing "1" on the keyboard switches the active window's stream to Norris' OBC.

Though it makes me sound kind of lazy it would be useful, especially during quali, to be able to assign the 0-9 keys to any stream the user would like. Typically during quali, I'll only have four OBCs open, and instead of constantly right-clicking and going through the context menu to switch a driver's stream it'd be easier to press a number key to switch the active window to an assigned stream. 

Bonus points if a modifier key plus a number could also be used, allowing for 20 total assignable hotkeys (0-9 and SHIFT+0-9).

**Comment by Kirito6B** [comment]
Keyboard shortcut are sometimes crowded with other functions. But could work to have something like a streamdeck. Allowing us to have the buttons we need in the layout we want.

**Reply by f1multiviewer** [reply]
There's already a streamdeck plugin (in the #integrations channel on our [Discord](https://muvi.gg/discord)) that you can use for this.
